source/pi/warble.sh

```diff
@@ -7,7 +7,24 @@ TWEET=$4
 
 pushd ../warble
 
-OUTPUT=$(python3 warblecc.py --username ${USERNAME} ${CODE})
+GRAALVISOR_IP=127.0.0.1
+# Checking if the default port of GraalVisor is in use.
+GRAALVISOR_UP=$(sudo lsof -i -P -n | grep LISTEN | grep 8080)
+if [ -z "$GRAALVISOR_UP" ]
+then
+  # GraalVisor is down, we have to launch it and register the function.
+  source ../pi/graalvisor_functions.sh
+  check_environment
+  start_polyglot_svm &> "$GRAALVISOR_PATH".log &
```

Comment:
There are two `&` signs (one here and another in start_svm), is this correct?

Reply:
Most probably that was a leftover from my previous changes and experiments. I merged the functions, now we only call `start_polyglot_svm` in background.